tests/unit/django_spanner/models.py

```diff
@@ -0,0 +1,67 @@
+# Copyright 2020 Google LLC
+#
+# Use of this source code is governed by a BSD-style
+# license that can be found in the LICENSE file or at
+# https://developers.google.com/open-source/licenses/bsd
+"""
+Different models used for testing django-spanner code.
+"""
+import os
+from django.db import models
+import django
+
+# Load django settings before loading dhango models.
+os.environ["DJANGO_SETTINGS_MODULE"] = "tests.settings"
+django.setup()
```

Comment:
Would using [`SimpleTestCase.settings`](https://docs.djangoproject.com/en/3.2/topics/testing/tools/#django.test.SimpleTestCase.settings) let you avoid clobbering the settings here?

Reply:
SimpleTestCase came with some overhead of setup and cleaning, got rid of it and using `unittest.TestCase` now.